manifest.ini

```diff
@@ -1,6 +1,20 @@
 [stage]
 reader_server = https://kinto.stage.mozaws.net/v1
 writer_server = https://kinto-writer.stage.mozaws.net/v1
+server = https://kinto.stage.mozaws.net/v1/
+version_fields = source, name, commit, version, build
+heartbeat_fields = storage, cache, permission
 
-[prod]
+[settings-prod]
 reader_server = https://firefox.settings.services.mozilla.com/v1/
```

Comment:
Is there a case where reader_server and server have different values? Or reader_server doesn't exists?

Should we try to read reader_server and fallback to server if it doesn't exist in the env?